e2e/cypress/integration/ui/settings/04_team_details.spec.ts

```diff
@@ -0,0 +1,237 @@
+describe('team management', () => {
+  let adminToken = '';
+  const now = Cypress.moment().format('MMDDYYhhmm');
+  const cypressPrefix = 'cypress-test';
+  const teamName = `${cypressPrefix} team ${now}`;
```

Comment:
this feels like a huge block of variables. can we move some down closer to the tests where they're actually used?

Reply:
I try to do that in general. Unfortunately I think all the ones listed above are used in both the beforeEach block and across different tests.